Libs/MRML/Core/vtkMRMLMeasurement.cxx

```diff
@@ -152,6 +152,54 @@ void vtkMRMLMeasurement::Copy(vtkMRMLMeasurement* src)
 #endif
 }
 
+//----------------------------------------------------------------------------
+void vtkMRMLMeasurement::SetEnabled(bool enabled)
+{
+  vtkDebugMacro(<< this->GetClassName() << " (" << this << "): setting Enabled to " << enabled);
+  if (this->Enabled != enabled)
+  {
+    this->Enabled = enabled;
+    this->Modified();
+    this->InvokeEvent(InputDataModifiedEvent);
+  }
+}
+
+//----------------------------------------------------------------------------
+void vtkMRMLMeasurement::EnabledOn()
+{
+  this->SetEnabled(true);
+}
+
+//----------------------------------------------------------------------------
+void vtkMRMLMeasurement::EnabledOff()
+{
+  this->SetEnabled(false);
+}
+
+//----------------------------------------------------------------------------
+void vtkMRMLMeasurement::SetUnits(std::string units)
+{
+  vtkDebugMacro(<< this->GetClassName() << " (" << this << "): setting Units to " << units);
+  if (this->Units != units)
+  {
+    this->Units = units;
+    this->Modified();
+    this->InvokeEvent(InputDataModifiedEvent);
+  }
+}
+
+//----------------------------------------------------------------------------
+void vtkMRMLMeasurement::SetPrintFormat(std::string format)
+{
+  vtkDebugMacro(<< this->GetClassName() << " (" << this << "): setting PrintFormat to " << format);
+  if (this->PrintFormat != format)
+  {
```

Comment:
```suggestion
    {
```

Reply:
Oops, right! Coming up.

Reply:
By the way we should really switch to aligned brackets...

Reply:
Fix done.